content/en/profiler/enabling/supported_versions.md

```diff
@@ -0,0 +1,48 @@
+| Minimum&nbsp;runtime&nbsp;version | JDK&nbsp;8+ | Python&nbsp;2.7+ | Go&nbsp;1.12+ | Ruby&nbsp;2.3+ | Node&nbsp;14+ | .NET&nbsp;Core&nbsp;2.1+, .NET&nbsp;5+, .NET&nbsp;Framework&nbsp;4.6.1+ | PHP&nbsp;7.1+ |  |
+| Recommended runtime version | JDK&nbsp;11+ | Python&nbsp;3.6+ | Go&nbsp;1.12+ | Ruby&nbsp;2.3+ | Node&nbsp;14+ | .NET&nbsp;6+ | PHP&nbsp;7.1+ | |
+| Minimum tracer version   | | 0.35.0 | 1.23.0 | 0.48.0 | 0.23.0 | 2.7.0| | |
+| Recommended tracer version |    | 0.50.0  | 1.51.0 | 1.15.0 | 2.12.0 | 2.31.0  | 0.92.0 | |
+| **Profile types:** |
+| {{< ci-details title="CPU" >}}The time each function/method spent running on the CPU.{{< /ci-details >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} |   | tracer&nbsp;2.15.0 | {{< X >}} | beta<br>ddprof&nbsp;0.1.0 |
+| {{< ci-details title="Allocation" >}}Number and sizes of memory allocations made by each function/method, including allocations which were subsequently freed.{{< /ci-details >}}   | JDK&nbsp;11+ | Python 3.6+<br>tracer&nbsp;0.50.0 | tracer&nbsp;1.47.0 |      |       | beta, .NET 6+<br>tracer&nbsp;2.18.0 | tracer&nbsp;0.88.0 | beta<br>ddprof&nbsp;0.9.3 |
+| {{< ci-details title="Heap" >}}The amount of heap memory allocated that remains in use.{{< /ci-details >}}   | JDK&nbsp;11+ | Python 3.6+<br> tracer&nbsp;0.50.0 | {{< X >}} |      | {{< X >}} | beta, .NET 6+<br>tracer&nbsp;2.22.0 |       | beta<br>ddprof&nbsp;0.15.0 |
+| {{< ci-details title="Locks" >}}The time each function/method spent waiting for and holding locks, and the number of times each function acquired a lock.{{< /ci-details >}}   | {{< X >}} | tracer&nbsp;0.45.0 | tracer&nbsp;1.47.0 |      |       | .NET 6+<br>tracer&nbsp;2.31.0 |       |      |
+| {{< ci-details title="Wall time" >}}The elapsed time spent in each function/method. Elapsed time includes time when code is running on CPU, waiting for I/O, and anything else that happens while the function/method is running.{{< /ci-details >}}   | {{< X >}} | {{< X >}} |       | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} |       |
+| {{< ci-details title="I/O" >}}The time each method spent reading from and writing to files and sockets.{{< /ci-details >}}   | {{< X >}} |       |       |       |       |       |       |       |
+| {{< ci-details title="Exceptions" >}}The number of exceptions raised, including those caught.{{< /ci-details >}}   | {{< X >}} | Python 3.7+ |       |       |       | .NET 5+<br>tracer&nbsp;2.31.0 |  beta<br>tracer&nbsp;0.92.0  |       |
+| **Other features:** |
+| {{< ci-details title="Code Hotspots" >}}Find specific lines of code related to performance issues.{{< /ci-details >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | beta | {{< X >}} | tracer&nbsp;0.71.0 |      |
+| {{< ci-details title="Endpoint Profiling" >}}Identify endpoints that are bottlenecks or responsible for heavy resource consumption.{{< /ci-details >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}} | beta | tracer&nbsp;2.15.0 | tracer&nbsp;0.79.0 |      |
```

Comment:
For Go, Code Hotspots and Endpoint Profiling were added in client library version 1.37.0